packages/vats/decentral-test-vaults-config.json

```diff
@@ -169,9 +169,6 @@
     "bridge": {
       "sourceSpec": "@agoric/vats/src/vat-bridge.js"
     },
-    "centralSupply": {
-      "sourceSpec": "@agoric/vats/src/centralSupply.js"
```

Comment:
:dart:

Reply:
oops! it needs to be gone from the devnet config too:

```
decentral-core-config.json:    "centralSupply": {
decentral-core-config.json:      "sourceSpec": "@agoric/vats/src/centralSupply.js"
decentral-demo-config.json:    "centralSupply": {
decentral-demo-config.json:      "sourceSpec": "@agoric/vats/src/centralSupply.js"
decentral-devnet-config.json:    "centralSupply": {
decentral-devnet-config.json:      "sourceSpec": "@agoric/vats/src/centralSupply.js"
```

ideally core too, but that's not critical

Reply:
removed from core-config and devnet-config, but left in demo-config